esrally/chart_generator.py

```diff
@@ -1195,21 +1330,40 @@ def load_track(cfg, name=None):
     return track.load_track(cfg)
 
 
-def generate_index_ops(chart_type, race_configs, environment):
+def generate_index_ops(chart_type, race_configs, environment, logger):
     idx_race_configs = list(filter(lambda c: "indexing" in c.charts, race_configs))
+    for race_conf in idx_race_configs:
+        logger.debug("Gen index visualization for race config with name:[{}] / label:[{}] / flavor: {} / license: {}".format(
```

Comment:
Our convention is to use plain logger strings for log statements instead of the format API.